pkg/streamingpromql/operators/functions/function_over_range_vector.go

```diff
@@ -179,7 +179,13 @@ func (m *FunctionOverRangeVector) NextSeries(ctx context.Context) (types.Instant
 
 func (m *FunctionOverRangeVector) emitAnnotation(generator types.AnnotationGenerator) {
 	metricName := m.metricNames.GetMetricNameForSeries(m.currentSeriesIndex)
-	m.Annotations.Add(generator(metricName, m.Inner.ExpressionPosition()))
+	pos := m.Inner.ExpressionPosition()
```

Comment:
perhaps `types.EmitAnnotationFunc` could take in a parameter for argument index?

Reply:
The difficulty with this is that `FunctionOverRangeVector` doesn't know which order the arguments were in originally (inner operator first, or scalars first?), so I think what's here is the least-confusing option.